### ISSUE TITLE
Startup intro message is gone in a flash

### ISSUE THREAD

**Issue by hexium310** [neovim/neovim#27612]
### Problem

In https://github.com/neovim/neovim/commit/eef2aedff6313d85b852463c78614ace57bb6f56 or later with tmux and [lualine.nvim](https://github.com/nvim-lualine/lualine.nvim), the startup intro message is gone in a flash. This issue doesn't appear outside tmux or inside tmux without `window-{,active-}style`.

tmux version: 3.4
lualine version: https://github.com/nvim-lualine/lualine.nvim/commit/7d131a8d3ba5016229e8a1d08bf8782acea98852

### Steps to reproduce

1. `tmux -f minimal.conf`
2. `nvim --clean -u minimal.lua`

`minimal.conf`
```
setw -g window-style bg='#393939'
setw -g window-active-style bg='#2d2d2d'
```

`minimal.lua`
```lua
local tmpdir = vim.uv.os_tmpdir()
local plugin_dir = vim.fs.joinpath(tmpdir, 'nvim_intro_is_gone')

if not vim.uv.fs_stat(plugin_dir) then
  vim.fn.system({
    'git',
    'clone',
    'https://github.com/nvim-lualine/lualine.nvim',
    plugin_dir,
  })
end

vim.opt.runtimepath:append(plugin_dir)

require('lualine').setup()
```

### Expected behavior

The startup intro message should still be displayed.

### Neovim version (nvim -v)

NVIM v0.10.0-dev-2440+g8addd2750

### Vim (not Nvim) behaves the same?

no

### Operating system/version

Arch Linux on WSL

### Terminal name/version

alacritty 0.13.1 (fe2a3c56)

### $TERM environment variable

tmux-256color (xterm-256color outside tmux)

### Installation

build from repo

**Comment by nichohells** [comment]
For me it doesn't display neither inside or outside Tmux, gone in a flash

**Comment by gpanders** [comment]
Possible duplicate of https://github.com/neovim/neovim/issues/27266.

**Comment by justinmk** [comment]
Duplicate of https://github.com/neovim/neovim/issues/3416

We should probably have some way for intro to redraw itself, instead of depending on a lack of redraws.

**Comment by zeertzjq** [comment]
Fixed by #27852
